submit/submit.cc

```diff
@@ -458,6 +459,37 @@ int main(int argc, char **argv)
 	return doAPIsubmit();
 }
 
+void curl_setup()
+{
+	handle = curl_easy_init();
+	if ( handle == NULL ) {
+		warning(0,"curl_easy_init() error");
+		return;
+	}
+
+	curl_easy_setopt(handle, CURLOPT_WRITEFUNCTION, writesstream);
+	curl_easy_setopt(handle, CURLOPT_ERRORBUFFER,   curlerrormsg);
+	curl_easy_setopt(handle, CURLOPT_FAILONERROR,   0);
+	curl_easy_setopt(handle, CURLOPT_FOLLOWLOCATION,1);
+	curl_easy_setopt(handle, CURLOPT_MAXREDIRS,     10);
+	curl_easy_setopt(handle, CURLOPT_NETRC,         CURL_NETRC_OPTIONAL);
+	curl_easy_setopt(handle, CURLOPT_TIMEOUT,       timeout_secs);
```

Comment:
Short answer: because it was that way before this PR.

Longer answer: Because it was added 13 years ago (62fe20560eadcabb2082af569e2dd2da0df7f7ac) when we used our own sockets implementation to connect directly to our own submitdaemon. Likely the implementation of libcurl added new constants, that were hardcoded, but the timeout constant was carried over becaue it already existed.